README.md

```diff
@@ -1,5 +1,138 @@
-## Work in Progress
+[![License](http://img.shields.io/:license-apache-blue.svg)](http://www.apache.org/licenses/LICENSE-2.0.html) [![Build Status](https://travis-ci.org/simp/pupmod-simp-ntpd.svg)](https://travis-ci.org/simp/pupmod-simp-ntpd) [![SIMP compatibility](https://img.shields.io/badge/SIMP%20compatibility-6.*-orange.svg)](https://img.shields.io/badge/SIMP%20compatibility-6.*-orange.svg)
 
-Please excuse us as we transition this code into the public domain.
+# pupmod-simp-ntpd
 
-Downloads, discussion, and patches are still welcome!
+#### Table of Contents
+
+1. [Description](#description)
+2. [Setup - The basics of getting started with ntpd](#setup)
+    * [What ntpd affects](#what-ntpd-affects)
+    * [Setup requirements](#setup-requirements)
+    * [Beginning with ntpd](#beginning-with-ntpd)
+3. [Usage - Configuration options and additional functionality](#usage)
+4. [Reference](#reference)
+5. [Limitations - OS compatibility, etc.](#limitations)
+6. [Development - Guide for contributing to the module](#development)
+
+## Description
+
+Set up ntpd in either standalone or server mode.
+
+## Setup
+
+
+### What ntpd affects
+
+Manages the following:
+
+* `ntp` package
+* `ntp` user/group
+* `ntpd` service
+* These files and directories:
+    * `/etc/ntp.conf`
+    * `/etc/ntp/`
+    * `/etc/ntp/keys`
+    * `/etc/sysconfig/ntpd`
+    * `/var/lib/ntp`
+
+### Setup Requirements
+
+This module requires the following:
+
+* [puppetlabs-concat](https://forge.puppet.com/puppetlabs/concat)
+* [puppetlabs-stdlib](https://forge.puppet.com/puppetlabs/stdlib)
+* [simp-auditd](https://forge.puppet.com/simp/auditd)
+* [simp-iptables](https://forge.puppet.com/simp/iptables)
+* [simp-simplib](https://forge.puppet.com/simp/simplib)
+
+### Beginning with ntpd
+
+## Usage
+
+    class { 'ntpd': }
+
+## Reference
+
+### Public Classes
+
+* [ntpd](https://github.com/simp/pupmod-simp-ntpd/blob/master/manifests/init.pp)
+
+#### Parameters
+
+* `servers` (`Variant[Array[String], Hash[String, Array[String]]]`) *(defaults to: `simplib::lookup('simp_options::ntpd::servers', { 'default_value' => {} })`)*
+
+An array of servers or a Hash of server/option pairs providing details for the NTP servers that this system should synchronize with
+
+**Example**
+
+```
+servers => {
+  'time.local.net' => ['iburst','minpoll 4', 'prefer'],
+  # This one will just use $default_options
+  'time.other.net' => []
+}
+```
+
+* stratum (`Integer[0]`) *(defaults to: `2`)*
```

Comment:
The formatting here makes it hard to find the list of parameters.  You might consider bolding them.

Reply:
typo-ed monitoring in line 104

Reply:
@judyj Fixed.

Reply:
@nick-markowski The parameters should have been `code`, but only the first one was.  They're now **`bold code`**.